pkgs/top-level/all-packages.nix

```diff
@@ -18580,8 +18580,22 @@ in
       bzip2 = null;
     };
   };
+  systemdMinimal = systemd.override {
```

Comment:
Could we also override the `name`  to be `systemd-minimal`?  This way people will be less confused when they see `systemd-v245` two times in their logs when downloading things from cache